lib/simpleofflinecache.js

```diff
@@ -5,9 +5,35 @@ var cacheHelper = require('sw-cache-helper');
 var debug = 0 ? console.log.bind(console, '[SimpleOfflineCache]') :
  function(){};
 
-function SimpleOfflineCache(cacheName) {
+// Default Match options, not exposed.
+var DEFAULT_MATCH_OPTIONS = {
+  ignoreSearch: false,
+  ignoreMethod: false,
+  ignoreVary: false
+};
+var DEFAULT_MISS_POLICY = 'fetchAndChace';
+// List of different policies
+var MISS_POLICIES = [
+  DEFAULT_MISS_POLICY
+];
+
+
+/**
+ * Constructor for the middleware that serves the content of a
+ * cache specified by it's name.
+ * @param {string} cacheName Name of the cache that will be serving the content
+ * @param {object} [options] Object use to setup the cache matching alternatives
+ * @param {string} [missPolicy] Name of the policy to follow if a request miss
+ *                 when hitting the cache.
+ */
+function SimpleOfflineCache(cacheName, options, missPolicy) {
   this.cacheName = cacheName || cacheHelper.defaultCacheName;
-  this.cache = null;
+  this.options = options || DEFAULT_MATCH_OPTIONS;
+  this.missPolicy = missPolicy || DEFAULT_MISS_POLICY;
```

Comment:
Still would like to keep it, and warning or not, not so sure about it, since it's in the spec. Chrome doesn't support it yet, but hopefully it will enter at some point.